lab_2/main.py

```diff
@@ -101,7 +98,21 @@ def calculate_distance(unknown_text_vector: list, known_text_vector: list) -> fl
     :param unknown_text_vector: vector for unknown text
     :param known_text_vector: vector for known text
     """
-    pass
+
+    if not isinstance(unknown_text_vector, list) or not \
+            isinstance(known_text_vector, list):
+        return None
+    for i in unknown_text_vector:
+        if not isinstance(i, (int, float)):
+            return None
+    for i in known_text_vector:
+        if not isinstance(i, (int, float)):
+            return None
+
+    distance = 0
+    for number, vector in enumerate(unknown_text_vector):
+        distance += (vector - known_text_vector[number])**2
+    return round(math.sqrt(distance), 5)
```

Comment:
what happens in this function?

Reply:
The distance is counted with euclid metric. At first, distance is equal to 0. For every vector in unknown_text_vector we add to distance the following: vector (from unknown_text_vector) - a vector from known_text_vector squared. Then the square root from this number is counted. We take only 5 digits of this number (with the round function).